.github/workflows/roborazzi_tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Roborazzi Integration Tests
+
+on:
+  push:
+    branches: [ master, 'robolectric-*.x' ]
+    paths-ignore:
+      - '**.md'
+
+  pull_request:
+#    branches: [ master, google ]
+    paths-ignore:
+      - '**.md'
+
+permissions:
+  contents: write # for uploading screenshots
+
+jobs:
+  graphics_tests:
```

Comment:
If we decide to go ahead with the automated update of the golden image logic, it might be better to use a separate workflow file. This is because it could get complicated. We should think about this once we've made a decision about the automated update.